agents/a2a/dispatcher.py

```
@@ -610,6 +615,51 @@ async def call_specialist(task: A2ATask) -> A2AResult:
+                    budget_spent=task.mandate.get("budget_spent", 0.0),
+                    iterations_used=task.mandate.get("iterations_used", 0),
+                    expires_at=_expires_at,
+                )
+                _mandate.record_invocation()
+                logger.debug(
+                    f"A2A: Recorded invocation for mandate '{_mandate.mandate_id}' "
+                    f"(iterations: {_mandate.iterations_used}/{_mandate.max_iterations}, "
+                    f"budget: {_mandate.budget_spent}/{_mandate.budget_limit})",
+                    extra={"mandate_id": _mandate.mandate_id}
+                )
+            except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)